api/api/views/media_views.py

```diff
@@ -147,6 +159,20 @@ def _validate_source(self, source):
                 detail=f"Invalid source '{source}'. Valid sources are: {valid_string}.",
             )
 
+    def include_addons(self, serializer):
```

Comment:
Are addons described anywhere in the codebase docs? Do we only have waveforms as addons now, or is there something else, too? 
I would like for this comment to include something like "addons, like waveforms for audio". A description of why something is an addon and not in the main model  (if it doesn't exist anywhere now) , would be very useful. Not necessarily in this PR, though :)

Reply:
The use of the addon model is specific to the audio media type and that too only for the waveforms. The `AudioAddOn` model does not have any docstrings, so the conversation around the PR that added this model https://github.com/WordPress/openverse-api/pull/510 is our only reference.

The PR describes that the [reason](https://github.com/WordPress/openverse-api/pull/510#pullrequestreview-877916062) the waveforms are in an `AudioAddOn` model is because the data in any columns that are not in the upstream DB would be gone after a data refresh.